meta/src/store/state_machine.rs

```diff
@@ -667,6 +671,37 @@ impl StateMachine {
         serde_json::to_string(&StatusResponse::default()).unwrap()
     }
 
+    fn process_add_date_node(&self, cluster: &str, node: &NodeInfo) -> CommandResp {
+        self.common_function(cluster, node)
+    }
+
+    fn change_node_state(&self, cluster: &str, node_id: &u64, node_state: &str) -> CommandResp {
+        let mut status = StatusResponse::new(META_REQUEST_FAILED, "".to_string());
+
+        if NodeState::from(node_state.to_owned()) == NodeState::Unknown {
```

Comment:
I see NodeState switching between enmu and string multiple times. Can you tell me what your intention is?

Reply:
In addition, semantic checking should detect problems earlier, not at the time of final execution

Reply:
I also don't want to convert because although multiple NodeStates have the same definition, they belong to different files, such as NodeState under meta_data.rs, some of which are in ast.rs or logical_ planner.rs. NodeState cannot be directly converted between them.Therefore, a medium String is required